IS_nRF24L01p/Class/nRF24L01p.vb

```diff
@@ -323,17 +324,24 @@ Public Class nRF24L01P
     ''' <summary>
     ''' 
     ''' </summary>
+    ''' <param name="SPIDesc">SPI bus descriptor. For RPi2 it's SPI0 or SPI1</param>
     ''' <param name="SPI"></param>
     ''' <param name="Speed"></param>
     ''' <param name="ChipEnablePin"></param>
     ''' <param name="InterruptPin"></param>
-    Public Sub New(SPI As SpiConnectionSettings, Speed As Integer, ChipEnablePin As Integer, InterruptPin As Integer)
+    Public Sub New(SPI As SpiConnectionSettings, Speed As Integer, ChipEnablePin As Integer, InterruptPin As Integer, Optional SPIDesc As String = "SPI0")
 
-        InitNordicSPI(SPI, Speed, ChipEnablePin, InterruptPin)
+        InitNordicSPI(SPIDesc, SPI, Speed, ChipEnablePin, InterruptPin)
 
     End Sub
 
-    Private Async Sub InitNordicSPI(SPI As SpiConnectionSettings, Speed As Integer, ChipEnablePin As Integer, InterruptPin As Integer)
+    Private Async Sub InitNordicSPI(SPIDesc As String, SPI As SpiConnectionSettings, Speed As Integer, ChipEnablePin As Integer, InterruptPin As Integer)
+
+        Dim strDesc As String = SpiDevice.GetDeviceSelector(SPIDesc)
+        Dim r As DeviceInformationCollection = Await DeviceInformation.FindAllAsync(strDesc)
+        If (r.Count = 0) Then
+            Return
```

Comment:
Instead of simply a return, we should throw an exception. Something like ; 
Throw New Exception("SPI port does not exist on the current system.")